### ISSUE TITLE
How to attach container to multiple docker network?

### ISSUE THREAD

**Issue by pshastri-dev** [fabric8io/docker-maven-plugin#1339]
Is there any way I can attach my container to two docker network? Docker does support that, I didn't find any solution from documentation to do such using fabric8 dmp.
I am using the latest version of dmp.

**Comment by rhuss** [comment]
Can you point me on a Docker example how this is supposed to work ?  Pretty sure that this plugin doesn't support this and tbh I very likely don't have time to work on this. But would be interested to understand the feature and used case. Also would help in integrating a PR

**Comment by pshastri-dev** [comment]
In docker networking, by default  a container will be created with one network attached but After the container has been created, more networks can be attached to a container using the docker network connect command. Have a look for an example: [https://success.docker.com/article/multiple-docker-networks](url)

**Comment by rhuss** [comment]
Nope, this is not support and probably won't be supported in the near future (except someone wants to jump on this issue to implement it)

**Comment by pshastri-dev** [comment]
Alright! Thank you for your help! :)
